opsdroid/logging.py

```diff
@@ -112,6 +113,8 @@ def configure_logging(config):
         with contextlib.suppress(KeyError):
             file_handler.addFilter(ParsingFilter(config, config["logging"]["filter"]))
 
+        rotation_handler = RotatingFileHandler(logfile_path, maxBytes=config["logging"].get('file-size', 50))
```

Comment:
Looks like `RotatingFileHandler` takes bytes not KB. So this will rotate at 50 bytes.

Let's bump this up, I recommend 50MB (`50e6`) would probably be a good default.

Reply:
Uh that was a fail haha I'll change that to 50mb